app/controllers/mentors_controller.rb

```diff
@@ -3,4 +3,9 @@ class MentorsController < ApplicationController
   def index
     @students = Student.all
   end
+
+  def show
```

Comment:
Interesting that you didn't update the index action and used the show action instead?